CloudSQLReplicator/google/cloud/pontem/sql/replicator/util/gcp_api_util.py

```diff
@@ -24,7 +24,16 @@
 
 from google.cloud.pontem.sql import replicator
 
-DEFAULT_TIER = 'db-n1-standard-2'
+
+STORAGE_SCOPE = ['https://www.googleapis.com/auth/devstorage.read_write']
```

Comment:
Is this a constant anywhere that you can depend on? If not, no worries. If it is, when we should use the constant.

Reply:
I have just seen the literal in code examples.